docs/how to/use-transferbench.rst

```diff
@@ -0,0 +1,22 @@
+.. meta::
+  :description: TransferBench documentation 
+  :keywords: TransferBench, API, ROCm, documentation, HIP
+
+
+Using TransferBench
+---------------------
+  
+Users have control over the SRC and DST memory locations by indicating memory type followed by the device index. TransferBench supports the following:
+
+* coarse-grained pinned host memory
+* unpinned host memory
+* fine-grained host memory
+* coarse-grained global device memory
+* fine-grained global device memory
+* null memory (for an empty transfer).
+
+In addition, users can determine the size of the transfer (number of bytes to copy) for their tests.
+
+Users can also specify executors of the transfer. The options are CPU, kernel-based GPU, and SDMA-based GPU (DMA) executors. TransferBench also provides the option to choose the number of sub-executors. In case of a CPU executor this argument specifies the number of CPU threads, while for a GPU executor it defines the number of compute units (CU). If DMA is specified as the executor, the sub-executor argument determines the number of streams to be used.
+
+Refer to :ref:`Examples` for a specific example.
```

Comment:
We only have one example. This bounces the user to a page with a single link. 
Let's get rid of the page and have a new link. 
Something like:
See ConfigFile Format for an example of using TransferBench.

Reply:
Done.

Reply:
Thanks, Leo. I added the example to the Use TransferBench file for contextual reference.